release/README-development-process.html

```diff
@@ -55,15 +55,29 @@ <h2 align="center">Checker Framework Development Process</h2>
                    projects in the future). If a performance regression was introduced, work with the feature owner to understand
                    whether bug fixes can fix the performance regression and to decide on the appropriate action.</li>
                    <li>Ensure that
-                   the changelogs for the Checker Framework, Type Annotations Compiler and AFU are complete.
-                   E-mail Mike asking him to vet the changelogs
+                   the changelogs and documentation for the Checker Framework, Type Annotations Compiler and AFU are complete by doing the following:
+                   <ol>
+                     <li>
+                               Log into Buffalo<br/><br/>
+                               <code>ssh $USER@buffalo.cs.washington.edu<br/></code></br>
+                     </li>
+                     <li>
+                               In a user-specific temporary directory, clone the Checker Framework repository (it contains the release scripts).<br/>
+                               If the repository exists, update it instead.<br/><br/>
+                               <code>mkdir -p /scratch/$USER/jsr308-release<br/></code>
+                               <code>cd /scratch/$USER/jsr308-release<br/></code>
```

Comment:
It's even easier to give the user a single set of commands to run, with no thinking required.  This works:

    mkdir -p /scratch/$USER/jsr308-release
    /scratch/$USER/jsr308-release
    git -C checker-framework pull || git clone https://github.com/typetools/checker-framework.git
    cd checker-framework/release